opo-makefile.mk

```diff
@@ -65,11 +65,11 @@ endif
 
 .PHONY: opo-test-e2e-up-local
 opo-test-e2e-up-local: opo-test-clean
-	operator-sdk test local ./test/e2e/ --up-local --namespace openshift-pipelines  --go-test-flags "-v -timeout=10m" --local-operator-flags "--recursive"
+	operator-sdk test local ./test/e2e/ --up-local --operator-namespace openshift-pipelines  --go-test-flags "-v -timeout=10m" --local-operator-flags "--recursive"
 
 .PHONY: opo-test-e2e
 opo-test-e2e: opo-test-clean
-	operator-sdk test local ./test/e2e/ --namespace openshift-operators  --go-test-flags "-v -timeout=10m" --local-operator-flags "--recursive"
+	operator-sdk test local ./test/e2e/ --operator-namespace openshift-operators  --go-test-flags "-v -timeout=10m" --local-operator-flags "--recursive"
```

Comment:
here only `--operator-namesapce` is requried :+1: 
as there is no `--up-local` flag, the `WATCH_NAMESPCE` var is set from the deployment manifest `deploy/operator.yaml`